entity/shaders/texture_blend_gaussian_blur.frag

```diff
@@ -0,0 +1,49 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+// 1D gaussian blur in the X direction.
+//
+// Paths for future optimization:
+//   * Remove the uv bounds check branch in SampleColor by adding optional
+//     support for SamplerAddressMode::ClampToBorder in the texture sampler.
+//   * Sample from higher mipmap levels when the blur radius is high enough.
+
+uniform sampler2D texture_sampler;
+
+in vec2 v_texture_coords;
+in vec2 v_texture_size;
+in vec2 v_blur_direction;
+in float v_blur_radius;
+
+out vec4 frag_color;
+
+const float TWO_PI = 6.283185307179586;
```

Comment:
Let's try to stick to C++ constant naming conventions? `kTwoPi`

Reply:
Done.